src/HDF5.jl

```diff
@@ -709,7 +709,7 @@ function h5read(filename, name::String; pv...)
     local dat
     fid = h5open(filename, "r"; pv...)
     try
-        obj = fid[name, pv...]
```

Comment:
Could we split this out to a separate PR  ?

Reply:
cc @kleinhenz  I think we missed this in the big update

Reply:
I thought about that... I agree.

Reply:
Done, see #651